src/cli/index.test.js

```diff
@@ -177,4 +177,10 @@ describe('cli', () => {
     expect(stdout).toMatchSnapshot()
     expect(shorthand).toEqual(stdout)
   })
+
+  it('--svg-attributes', async () => {
+    const [stdout] = await exec('bin/svgr --svg-attribute focusable=false --svg-attribute hidden=0 __fixtures__/one.svg')
```

Comment:
Look like it does not work, the snapshot does not reflect the change.

Reply:
S***t ! Github did not send me an email to say that the tests don't pass in the CI :s 

I do it !